pkg/vault/test/foundry/fork/ERC4626WrapperBase.t.sol

```diff
@@ -197,20 +212,90 @@ abstract contract ERC4626WrapperBaseTest is Test {
         );
     }
 
-    function _initializeUserWallet() private {
-        (user, ) = makeAddrAndKey("User");
-        vm.label(user, "User");
+    function testAddLiquidityToBuffer__Fork__Fuzz(
+        uint256 underlyingToInitialize,
+        uint256 wrappedToInitialize,
+        uint256 sharesToIssue
+    ) public {
+        underlyingToInitialize = bound(underlyingToInitialize, 1e6, underlyingToken.balanceOf(lp) / 10);
```

Comment:
Are these minimum numbers deliberate? They don't seem to correspond to any of the relevant constants (e.g., MIN_WRAP_AMOUNT, _BUFFER_MINIMUM_TOTAL_SUPPLY)